internal/keycloakb/configdbauthtransaction.go

```diff
@@ -0,0 +1,83 @@
+package keycloakb
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"strings"
+
+	"github.com/cloudtrust/common-service/v2/configuration"
+	"github.com/cloudtrust/common-service/v2/database/sqltypes"
+)
+
+// AuthorizationTransaction interface
+type AuthorizationTransaction interface {
+	Close() error
+	Commit() error
+
+	CreateAuthorizations(ctx context.Context, authz []configuration.Authorization) error
+	RemoveAuthorizations(ctx context.Context, authz []configuration.Authorization) error
+}
+
+type authorizationTransaction struct {
+	tx sqltypes.Transaction
+}
+
+// NewAuthorizationTransaction creates a new AuthorizationTransaction
+func NewAuthorizationTransaction(tx sqltypes.Transaction) AuthorizationTransaction {
+	return &authorizationTransaction{tx: tx}
+}
+
+func (at *authorizationTransaction) Close() error {
+	return at.tx.Close()
+}
+
+func (at *authorizationTransaction) Commit() error {
+	return at.tx.Commit()
+}
+
+func (at *authorizationTransaction) executeQuery(query string, args ...any) error {
+	_, err := at.tx.Exec(query, args...)
+	return err
+}
+
+func (at *authorizationTransaction) CreateAuthorizations(ctx context.Context, authz []configuration.Authorization) error {
+	// 50 rows per actual DB request
+	return at.createAuthorizationsMax(ctx, authz, 50)
```

Comment:
Should the max row be a parameter that we manually want to tweak (in config.yml file)? In order to not change the code if we want to change that.